front/app/components/admin/AddCollaboratorsModal/index.tsx

```diff
@@ -64,19 +60,9 @@ const AddCollaboratorsModal = ({
         <Box>
           <Text color="textPrimary" fontSize="m" my="0px">
             {hasReachedLimit ? (
-              <FormattedMessage
-                {...messages.reachedLimitText}
-                values={{
-                  noOfSeats: noOfCollaboratorSeatsToAdd,
-                }}
-              />
+              <FormattedMessage {...messages.reachedLimitText} />
```

Comment:
Since the condition is on the message and the values passed through are not different, we could do this
```suggestion
              <FormattedMessage {...(hasReachedLimit ? messages.reachedLimitText : messages.confirmMessage)} />
```